controllers/grid/issues/ExportableIssuesListGridHandler.inc.php

```diff
@@ -42,8 +44,18 @@ public function getSelectName()
     protected function loadData($request, $filter)
     {
         $journal = $request->getJournal();
-        $issueDao = DAORegistry::getDAO('IssueDAO'); /* @var $issueDao IssueDAO */
-        return $issueDao->getIssues($journal->getId(), $this->getGridRangeInfo($request, $this->getId()));
+
+        // Handle grid paging (deprecated style)
+        $rangeInfo = $this->getGridRangeInfo($request, $this->getId());
+        $collector = Repo::issue()->getCollector()
+            ->filterByContext($journal->getId());
+        $totalCount = Repo::issue()->getCount($collector);
+        $collector->limit($rangeInfo->getCount());
+        $collector->offset($rangeInfo->getoffset() + max(0, $rangeInfo->getPage() - 1) * $rangeInfo->getCount());
```

Comment:
Is `$rangeInfo->getoffset()` supposed to be all lower case?

Reply:
Nope. Should be `getOffset()`. Updated.